package.json

```diff
@@ -81,8 +82,6 @@
     "html-looks-like": "^1.0.2",
     "lodash": "^4.17.4",
     "ncp": "^2.0.0",
-    "node-sass": "^4.5.3",
```

Comment:
I've added them for sass tests - no longer needed.

Doesn't matter now coz there is a bug in npm 5 which causes "devDepdencies" to be included when installing locally.